wordpress-mobile/WordPress-Android
# Fix stats crash

Fixes #9582. This PR is supposed to prevent any future crash due to uninitialized variable (it's now initialized on object creation). It adds a few more preventive measures, namely passing around site ID instead of a serialized `SiteModel`.

To test:
1. Go to Notifications tab
2. Tap Follows tab
3. Scroll to bottom of list
4. Tap See all of your followers button link (the site must have 10+ followers for this to appear)
5. Notice a list of all followers is loaded
